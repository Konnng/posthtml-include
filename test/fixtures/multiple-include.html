<include src="./test/fixtures/includes/6.html">{ "localVariable": "local1" }</include>
<include src="./test/fixtures/includes/6.html">{ "localVariable": "local2" }</include>
<include src="./test/fixtures/includes/6.html">{ "localVariable": "local3" }</include>
<include src="./test/fixtures/includes/6.html">{ "localVariable": "local4" }</include>
<include src="./test/fixtures/includes/6.html">{ "localVariable": "local5" }</include>
