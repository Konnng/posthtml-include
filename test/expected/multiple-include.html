<div>local1</div>
<div>global</div>

<div>local2</div>
<div>global</div>

<div>local3</div>
<div>global</div>

<div>local4</div>
<div>global</div>

<div>local5</div>
<div>global</div>
